backend/tracim_backend/models/data.py

```diff
@@ -1422,12 +1434,17 @@ def format_path(cls, url_template: str, content: "Content") -> str:
         cid = content.content_id
         return url_template.format(wid=wid, fid=fid, ctype=ctype, cid=cid)
 
-    def copy(self, parent):
-        cpy_content = Content()
-        for rev in self.revisions:
-            cpy_rev = ContentRevisionRO.copy(rev, parent)
-            cpy_content.revisions.append(cpy_rev)
-        return cpy_content
+    @property
+    def recursive_tree_revision(self):
+        """Get all revision sorted by id of content and all his children recursively"""
+        revisions = []  # type: typing.List[ContentRevisionRO]
+        for revision in self.revisions:
+            revisions.append(revision)
+        for children in self.recursive_children:
+            for revision in children.revisions:
```

Comment:
replace:

````
    for children in self.recursive_children:
        for revision in children.revisions:
            revisions.append(revision)
````
by:

````
    for chil in self.recursive_children:
        revisions.extend(child.get_tree_revisions())
````

note : children is plura form of child. A loop must be `for child in self.children`

Reply:
```
            for child in self.get_children(recursively=True):
                revisions.extend(child.revisions)
```
is more correct. we already get all children recursively, it doesn't make sens to get for each of them recursive tree revisions.

Reply:
ok

Reply:
done.